userspace/libsinsp/parsers.cpp

```diff
@@ -5244,6 +5233,27 @@ void sinsp_parser::parse_container_json_evt(sinsp_evt *evt)
 		}
 		evt->m_tinfo_ref = container_info->get_tinfo(m_inspector);
 		evt->m_tinfo = evt->m_tinfo_ref.get();
+		container_info->set_lookup_status(static_cast<sinsp_container_lookup::state>(lookup_state.asUInt()));
+		switch(container_info->get_lookup_status())
+		{
+		// Preserve case where lookup_state == STARTED from scap file was set to FAILED
+		// or otherwise set to false.
+		case sinsp_container_lookup::state::FAILED:
+			break;
+		default:
+			{
+				if (!container_info->m_image.empty())
+				{
+					// Marked as SUCCESSFUL if and only if the container image field present.
+					container_info->set_lookup_status(sinsp_container_lookup::state::SUCCESSFUL);
```

Comment:
I would propose to raise the bar here :upside_down_face:

Reply:
note to self: only do this enhanced check for actual containers not pod sandbox containers in the CRI cases, forgot about it here.